src/GPL-1.0.xml

```diff
@@ -0,0 +1,242 @@
+<spdx name="GNU General Public License v1.0 only" identifier="GPL-1.0" osi-approved="false">
+  <urls>
+    <url>http://www.gnu.org/licenses/old-licenses/gpl-1.0-standalone.html</url>
+  </urls>
+  <notes>This license was released: February 1989. This license has been deprecated. This refers to when this GPL 1.0
```

Comment:
in this case "deprecated" means by the author in lieu of a more recent version.  See https://spdx.org/spdx-license-list/license-list-overview for full description of the Notes field. Now that we have "deprecated" licenses in the SPDX License List, we might want to use different language there... although the explanations should clarify?